device/iot-device-samples/pom.xml

```diff
@@ -40,6 +40,11 @@
         <module>custom-sas-token-provider-sample</module>
     </modules>
     <dependencies>
+        <!--suppress MavenDuplicateDependenciesInspection -->
+        <!--
+        This duplicate dependency declaration warning can be ignored as one child Pom.xml duplicates this dependency declaration
+        demonstrate how to exclude in order to demonstrate excluding bouncycastle dependency from the device client library.
```

Comment:
I think there is a typo here: "...duplicates this dependency declaration ~demonstrate how to exclude~ in order to demonstrate excluding bouncycastle dependency"